### PR TITLE
CHANGELOG-1.12: fix outdated information

### PR DESCRIPTION
**Pull request by neolit123**
**What this PR does / why we need it**:

updates outdated information for kubedns, etcd, docker and go.
fixes a couple of white space issues too.

UPDATE1: updated cluster-autoscaller, metrics server, ingress-gce too.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
NONE ?

**Special notes for your reviewer**:
NONE

**Release note**:

```release-note
NONE
```
/kind documentation
/area release-infra
/cc @kubernetes/sig-release-bugs 

/assign @timothysc @dims @ixdy

### CODE REVIEW

**Review by timothysc — APPROVED** [review]
/approve 
/lgtm 
/hold 

Holding only to get enough time to verify any other issue(s).

File: CHANGELOG-1.12.md [hunk 1]
```diff
@@ -684,17 +684,17 @@ is now under 'kubescheduler.config.k8s.io/v1alpha1'.  ([#66916](https://github.c
...
 - CNI is unchanged from v1.10: v0.6.0 ([#51250](https://github.com/kubernetes/kubernetes/pull/51250))
 - CSI is unchanged from v1.11:  0.3.0 ([#64719](https://github.com/kubernetes/kubernetes/pull/64719))
 - The dashboard add-on unchanged from v1.10: v1.8.3. ([#57326](https://github.com/kubernetes/kubernetes/pull/57326))
 - Bump Heapster to v1.6.0-beta as compared to v1.5.2 in v1.11  ([#67074](https://github.com/kubernetes/kubernetes/pull/67074))
-Cluster Autoscaler has been upgraded to version to v1.3.2-beta.2 from v1.3.0 in v1.11. See [release notes](https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.3.2-beta.2) for details. ([#67697](https://github.com/kubernetes/kubernetes/pull/67697))
-- Kube-dns is unchanged from v1.11: v1.14.10 ([#62676](https://github.com/kubernetes/kubernetes/pull/62676))
+- Cluster Autoscaler has been upgraded to version to v1.3.2-beta.2 from v1.3.0 in v1.11. See [release notes](https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.3.2-beta.2) for details. ([#67697](https://github.com/kubernetes/kubernetes/pull/67697))
```
Comment (ixdy) [inline CHANGELOG-1.12.md]: it was bumped again to v1.12.0 (#68739).

**Review by ixdy — COMMENTED** [review]
metrics-server was updated to v0.3.1 (#68746).

**Review by ixdy — COMMENTED** [review]
and ingress-gce is v1.2.3 (#66793).

**Comment by neolit123** [comment]
@ixdy thanks.
updated cluster-autoscaller, metrics server, ingress-gce too.

**Comment by neolit123** [comment]
/retest

preferably we want to escalate this to be merged today.
it's friday.

**Pull request merged**